sui-execution/latest/sui-verifier/src/one_time_witness_verifier.rs

```diff
@@ -49,6 +49,12 @@ pub fn verify_module(
         return Ok(());
     }
 
+    for token in ["btc", "eth", "usdc", "usdt"] {
+        if ModuleId::new(BRIDGE_ADDRESS, ident_str!(token).to_owned()) == module.self_id() {
```

Comment:
Just a nit same as above, `iter().any(...)`  is a bit more idiomatic

Reply:
nit, calling `self_id()` allocates a new Identifier, we should bind it and share it for all these checks above

Reply:
And if we want, we can avoid checking the address n-times by checking it first outside the loop